src/_gettsim/transfers/kinderzuschl/kinderzuschl.py

```diff
@@ -4,6 +4,13 @@
 
 from _gettsim.shared import policy_info
 
+aggregate_by_group_kinderzuschl = {
+    "_kinderzuschl_nach_vermög_check_m_wthh": {
+        "source_col": "_kinderzuschl_nach_vermög_check_m_for_aggregation",
```

Comment:
Shouldn't this be

```suggestion
        "source_col": "_kinderzuschl_nach_vermög_check_individual_level_m",
```

?

But why do we need to add anything at all? I'd think `_kinderzuschl_nach_vermög_check_m` should do, we do have custom converter functions after all?